spec/features/user/user_views_past_orders_spec.rb

```diff
@@ -0,0 +1,140 @@
+require 'rails_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.